### PR TITLE
New Port: Trigger Rally (Racing)

### PR DESCRIPTION
**Pull request by Cebion**
Game Information
Title: Trigger Rally
URL: https://github.com/Cebion/trigger-rally_pm

Mouse is invisible but can be navigate trough hovering over the menu items.

## Controls

| Button | Action |
|--|--| 
|DPAD| Steer|
|A| Accelerate|
|B| Brake|
|Y| Handbrake|
|X| Change Camera View|
|L1 | Map|
|L2| Toggle UI|
|R1 | Left Mouse|
|R2 | Recover Car|
|L3 | Pause|

CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] ROCKNIX
[X] MuOS
[] Knulli (Optional)
[] Crossmix (Optional)

Architectures: 
[x] Aarch64
[X] Panfrost (Mainline)

Resolutions: 
[] 480x320 (Optional)
[X] 640x480
[X] 720x720 (RGB30) (Optional)
[X] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/triggerrally/Trigger Rally.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# PORTMASTER: triggerrally.zip, Trigger Rally.sh
```
Comment (Slayer366) [inline ports/triggerrally/Trigger Rally.sh:2]:
```suggestion
```
Previously run PM entry.

**Review by Slayer366 — APPROVED** [review]

File: ports/triggerrally/gameinfo.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <name>Trigger Rally</name>
```
Comment (cdeletre) [inline ports/triggerrally/gameinfo.xml:4]:
```suggestion
    <name>Trigger Rally</name>
    <path>Trigger Rally.sh</path>
```

**Review by cdeletre — CHANGES_REQUESTED** [review]
That's the best I can check :D

**Review by Slayer366 — DISMISSED** [review]

**Review by cdeletre — APPROVED** [review]

File: ports/triggerrally/gameinfo.xml (hunk 2 above)
Comment (Slayer366) [reply ports/triggerrally/gameinfo.xml:4]: Upon second look, should this not have been:
```
<path>./Trigger Rally.sh</path>
```

**Pull request merged**